cmd/topicctl/subcmd/apply.go

```diff
@@ -197,9 +197,12 @@ func applyTopic(
 		adminClient, err = clusterConfig.NewAdminClient(
 			ctx,
 			nil,
-			applyConfig.dryRun,
-			applyConfig.shared.saslUsername,
-			applyConfig.shared.saslPassword,
+			config.AdminClientOpts{
```

Comment:
I refactored `NewAdminClient` to take in a struct of `AdminClientOpts` instead of continuing our anti-pattern of passing in an ever growing list of configuration options as separate parameters. There are many instances of this change in this PR